pennylane/ops/op_math/pow.py

```diff
@@ -339,6 +337,11 @@ def adjoint(self):
         return Pow(base=qml.adjoint(self.base), z=self.z)
 
     def simplify(self) -> Union["Pow", Identity]:
+        # try using pauli_rep:
+        if pr := self._pauli_rep:
+            pr.simplify()
+            return pr.operation(wire_order=self.wires)
```

Comment:
Not sure we want to use this logic for `Pow`.  Does pauli rep simplification cover cases like `PauliX(0) ** 0.5`?

Reply:
In the init for pow the pauli representation is only defined if the exponent is a positive integer :)

Reply:
Ah ok. thanks for the clarification.

Reply:
Maybe leave a comment in the source code here?